### PR TITLE
Moved code section of identity to table format

### LINKED ISSUE #5668: [Sistent] Move `code` section of identity to table format
### Current Behavior

The identity section under https://layer5.io/projects/sistent/identity/color like color, typography and spacing these sections code part is currently a image so user can't copy the color code.

### Desired Behavior

Move these images to table format instead of images, same structure just use table to display the colors and rest code section of identity.
Design can be found here: [Figma](https://www.figma.com/design/ekm0Hh6M25bSKpP5BcIJom/Sistent-Design-System?node-id=1379-19519&t=mFkghFS6lrXZEMTg-0)

### Acceptance Tests

---
<h2>Contributor Resources and <a href="https://layer5.io/community/handbook">Handbook</a></h2>

The layer5.io website uses Gatsby, React, and GitHub Pages. Site content is found under the [`master` branch](https://github.com/layer5io/layer5/tree/master).
- 📚 See [contributing instructions](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md).
-  🎨 Wireframes and [designs for Layer5 site](https://www.figma.com/file/5ZwEkSJwUPitURD59YHMEN/Layer5-Designs) in Figma [(open invite)](https://www.figma.com/team_invite/redeem/qJy1c95qirjgWQODApilR9)
- 🙋🏾🙋🏼 Questions: [Discussion Forum](https://discuss.layer5.io) and [Community Slack](https://slack.layer5.io).

Join the Layer5 Community by submitting your [community member form](https://layer5.io/newcomer).

### PR DESCRIPTION
**Pull request by BarryByte**
**Description**
This PR fixes #5668 [Sistent] Move code section of identity to table format.

@sudhanshutech @Ashparshp @narasimha-1511  Please review this pull request. If you have any suggestion please let me know!

**[Signed commits](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by l5io** [comment]
🚀 Preview for commit 1dc9115cac16f744be9ff77afee641f1fa9ecf0f at: https://66b63c801c5cc21dde094da5--layer5.netlify.app

File: src/sections/Projects/Sistent/identity/color/code.js [hunk 1]
```diff
@@ -26,6 +25,1029 @@ import { useStyledDarkMode } from "../../../../../theme/app/useStyledDarkMode";
 const ColorCode = () => {
   const { isDark } = useStyledDarkMode();
   const location = useLocation();
+  const getColorColumns = (isPlusRole) => {
+    if (isPlusRole) {
+      return ["Token", "Value", "Description"];
+    }
+    return ["Token", "Value"];
+  };
+  const colorData = [
```
Comment (narasimha-1511) [inline src/sections/Projects/Sistent/identity/color/code.js:34]: make this const data in a separate file an import it or , do some thing , should not declare constants here as it is very large

File: src/sections/Projects/Sistent/identity/color/code.js [hunk 2]
```diff
@@ -26,6 +25,1029 @@ import { useStyledDarkMode } from "../../../../../theme/app/useStyledDarkMode";
...
+      token: "background-error-tertiary",
+      value: {
+        name: "Alias - red-70",
+        hex: "#f2e3da",
+        swatchClass: "background-error-tertiary",
+      },
+      role: "Second level error surface color which is used to establish prominence and layer hierarchy.",
+    },
+  ];
+  const TextColors = [
```
Comment (narasimha-1511) [inline src/sections/Projects/Sistent/identity/color/code.js:728]: replace this also

**Review by narasimha-1511 — CHANGES_REQUESTED** [review]

**Comment by Ashparshp** [comment]
@BarryByte Thanks for your contribution, let's discuss this on the website's call. Please add this as an agenda item to the [meeting minutes](https://docs.google.com/document/d/1XczAHXVe2FIWPqiF57ospJ43zw5cZQ7ui8mn39v5EvA/edit?usp=drivesdk).

File: src/sections/Projects/Sistent/identity/color/code.js (hunk 1 above)
Comment (BarryByte) [reply src/sections/Projects/Sistent/identity/color/code.js]: resolved

File: src/sections/Projects/Sistent/identity/color/code.js (hunk 2 above)
Comment (BarryByte) [reply src/sections/Projects/Sistent/identity/color/code.js]: resolved

**Comment by l5io** [comment]
🚀 Preview for commit 2534de7a6f58a39db84c73af6be512060c047c67 at: https://66b8c324a80f80562948a12d--layer5.netlify.app

**Comment by BarryByte** [comment]
@sudhanshutech I searched for similar tables in the website but everywhere there's an image inserted instead of table that was discussed in the meeting. Please provide link to similar tables, there is an change i might missed. Thanks.

**Comment by sudhanshutech** [comment]
here is the component: https://github.com/layer5io/sistent/blob/master/src/custom/ResponsiveDataTable.tsx
and for examples: https://github.com/meshery/meshery/blob/master/ui/components/connections/index.js#L1213

@BarryByte

**Comment by Aviral0702** [comment]
@BarryByte Any updates?

**Pull request closed without merging**